client/src/pages/OnboardingPage/OnboardingPage.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import { Link } from "@storybook/router";
+import UserSetupForm from "./UserSetupForm/UserSetupForm";
+import FilterSettingsForm from "./FilterSettingsForm/FilterSettingsForm";
+import PersonalSettingsForm from "./PersonalSettingsForm/PersonalSettingsForm";
+import InputTextField from "components/InputTextField/InputTextField";
+import Button from "components/Button/Button";
+import ProfilePicture from "components/ProfilePicture/ProfilePicture";
+import "./OnboardingPage.css";
+
+class OnboardingPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      counter: 1,
+      setup: {
+        name: '',
+        email: '',
+        password: ''
+      },
+      filter: {
+        city: '',
+        tags: []
+      }, 
+      personal: {
+        img: '',
+        age: 0,
+        destination: ''
+      }
```

Comment:
We're not really concerned with where in the form this data is (i.e. `setup`, `filter`, or `personal`). Instead, all this data is going to build out a user and their profile. So maybe we could nest these properties in an object called `profile` or `user`?